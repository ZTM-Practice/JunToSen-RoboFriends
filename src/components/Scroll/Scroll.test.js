import { shallow } from "enzyme";
import React from "react";
import Scroll from "./Scroll";

describe('Scroll Component', () => {
    it('it renders the Scroll component', () => {
        expect(shallow(<Scroll />)).toMatchSnapshot();
    });
});
