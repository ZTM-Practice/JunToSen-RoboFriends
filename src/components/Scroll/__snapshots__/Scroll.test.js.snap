// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll Component it renders the Scroll component 1`] = `ShallowWrapper {}`;
